dynawo/sources/Models/Modelica/PreassembledModels/CMakeLists.txt

```diff
@@ -83,6 +83,7 @@ set(MODELICA_PREASSEMBLED_MODELS
   InfiniteBusWithImpedance.xml
   InfiniteBusWithVariations.xml
   InjectorBG.xml
+  IntegerDoubleStep.xml
```

Comment:
Is there a reason why the classical DoubleStep cannot be used in your cases ? Why does it specifically require an integer ?

Reply:
In the Alstom and Siemens SVarC, setModeManual is an IntegerInput corresponding to one of three modes. I could replace it with a RealInput, but this would look weird.

Reply:
Ok thanks